agent/cache/cache.go

```diff
@@ -214,6 +214,44 @@ func (c *Cache) Get(t string, r Request) (interface{}, ResultMeta, error) {
 	return c.getWithIndex(t, r, r.CacheInfo().MinIndex)
 }
 
+// retrieves a cache entry and checks if it is ready to be returned given the other parameters
+func (c *Cache) getEntryLocked(tEntry typeEntry, key string, maxAge time.Duration, revalidate bool, minIndex uint64) (bool, bool, cacheEntry) {
+	entry, ok := c.entries[key]
+
+	if !ok {
+		return false, false, entry
```

Comment:
Nitpick to be more clear what is returned: return `ok` as the first value, instead of just `false`.